test/shopify-cli/core/monorail_test.rb

```diff
@@ -39,23 +40,41 @@ def test_log_doesnt_prompt_for_consent_if_already_answered
 
       def test_log_event_contains_schema_and_payload_values
         enabled_and_consented(true, true)
-        stub_request(:post, Monorail::ENDPOINT_URI).to_return(status: 200)
-
-        JSON.expects(:dump).with(
-          has_entries(
-            schema_id: ShopifyCli::Core::Monorail::INVOCATIONS_SCHEMA,
-            payload: has_entries(
-              cli_sha: "bb6f42193239a248f054e5019e469bc75f3adf1b",
-              uname: instance_of(String),
-              args: "testcommand arg argtwo",
-              timestamp: instance_of(String),
-              duration: instance_of(Float),
-              result: ShopifyCli::Core::Monorail::SUCCESS_SENTINEL
+        Timecop.freeze do |time|
```

Comment:
should we have test cases for/without the conditional addition of metadata and project data?